Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/DeclarationsSectionTranslator.swift

```diff
@@ -95,13 +186,16 @@ struct DeclarationsSectionTranslator: RenderSectionTranslator {
                     for alternateDeclaration in decls {
                         let renderedTokens = alternateDeclaration.declarationFragments.map(translateFragment)
 
-                        let platformNames = platforms
+                        let platformNames =
+                            platforms
                             .compactMap { $0 }
                             .sorted(by: \.rawValue)
 
                         declarations.append(
                             DeclarationRenderSection(
-                                languages: [trait.interfaceLanguage ?? renderNodeTranslator.identifier.sourceLanguage.id],
+                                languages: [
+                                    trait.interfaceLanguage ?? renderNodeTranslator.identifier.sourceLanguage.id
+                                ],
```

Comment:
Minor: This "languages" value is the same for the entire `translateSection(for:renderNode:renderNodeTranslator:)` implementation. You could compute it only once.